idx/memory/memory.go

```diff
@@ -274,7 +269,7 @@ func (m *MemoryIdx) Load(defs []schema.MetricDefinition) int {
 
 		m.add(def)
 
-		if tagSupport {
+		if TagSupport {
```

Comment:
should this be `if TagSupport && len(def.Tags) > 0 {` ?

Reply:
actually not, it should just be `if TagSupport` because `indexTags` also indexes the `name` as tag. But while looking at this i noticed that i'll need to fix the other call-site of `indexTags()` accordingly

Reply:
fixed